data/transit/route/bus.json

```
@@ -8775,7 +8801,11 @@
       "locationSet": {"include": ["fr"]},
       "tags": {
         "network": "Rubis",
-        "operator": "Rubis",
+        "network:wikidata": "Q16681031",
+        "network:wikipedia": "fr:Transports en commun de Bourg-en-Bresse",
+        "operator": "Keolis Grand Bassin de Bourg-en-Bresse",
```

Would it be ok to just use "Keolis" here?
```suggestion
        "operator": "Keolis",
```

---

same thing.
Keolis Grand Bassin de Bourg-en-Bresse is a subsidiary of Keolis and has no page on wikipedia and wikidata.